src/utils/manualTransaction.ts

```diff
@@ -0,0 +1,31 @@
+import { Connection, Keypair, PublicKey, Transaction } from "@solana/web3.js";
+
+export async function manualSendTransaction(
```

Comment:
please solve the issue with using anchor, also if transaction fails we should see a full log